### PR TITLE
add feature get all keys (object)

### LINKED ISSUE #87: オブジェクトの全キーを取得する
概要(Summary)
・オブジェクトのキーを取得するメソッドを作成
・allKeys() 
オブジェクトのキーを取得する
・Return
オブジェクトのキーが格納された配列
・対象
NCMBObject.java

### PR DESCRIPTION
**Pull request by hocnguyen**
## 概要(Summary)

- Fixed #87 

## 動作確認手順(Step for Confirmation)

Run the unit test.

### CODE REVIEW

**Comment by ndyuya** [comment]
I think that you should add "allKeys" method to NCMBBase instead of NCMBObject.
Because NCMBRole, NCMBFile and NCMBPush　do not be inherit NCMBObject. (extends NCMBBase)

**Comment by ykokw** [comment]
Thanks for creating pull request.

Could you fix to return Set or List instead of JSONArray?

**Comment by ykokw** [comment]
Could you use Gson library to convert List from JSONArray?
(It become more simply imprementation)
You could see other method where use Gson library in SDK.
https://github.com/NIFTYCloud-mbaas/ncmb_android/blob/dc1848208b3e47adee20136db90dd9b4e4d9a9d3/ncmb-core/src/main/java/com/nifty/cloud/mb/core/NCMBBase.java#L584

File: ncmb-core/src/main/java/com/nifty/cloud/mb/core/NCMBBase.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.nifty.cloud.mb.core;
 
 import android.location.Location;
+import android.util.Log;
```
Comment (hocnguyen) [inline ncmb-core/src/main/java/com/nifty/cloud/mb/core/NCMBBase.java]: Maybe when test, I used Log() and forget remove it.
Sorry about that.

Comment (ykokw) [inline ncmb-core/src/main/java/com/nifty/cloud/mb/core/NCMBBase.java]: OK, then could you remove it ? I'll merge this p-r.

**Pull request merged**